runestone/mchoice/js/mchoice.js

```diff
@@ -394,8 +394,9 @@ export default class MultipleChoice extends RunestoneBase {
             if (buttonObjs[i].checked) {
                 given = buttonObjs[i].value;
                 this.givenArray.push(given);
-                this.feedbackString += `<li value="${i + 1}">${this.feedbackList[i]
-                    }</li>`;
+                this.feedbackString += `<li value="${i + 1}">${
+                    this.feedbackList[i]
+                }</li>`;
```

Comment:
This block looks like it should be hand formatted.  Not sure what formatter thinks it is a good idea to do `${` on one line and `}` after it makes it really hard to see that it is jQuery.

Reply:
I'll revert this. I use [Prettier](https://prettier.io/) to format JS.

Reply:
But also, this isn't JQuery, it's a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).

Reply:
Here's the code before Prettier:

``` JavaScript
                this.feedbackString += `<li value="${i + 1}">${this.feedbackList[i]
                    }</li>`;
```

This seems worse than what Prettier did to it!

Reply:
So on second thought, let's leave this change in.